main.cpp

```diff
@@ -387,14 +390,23 @@ struct StudioLighting
 #include <iostream>
 int main()
 {   
-    MidiKeyboard nord;
-    nord.pitchShift();
-
-    Laptop macbook;
-    macbook.hardDriveDetails();
-
+    
+    MidiKeyboard keyboard;
+    keyboard.printKeys();
+    Laptop myLaptop;
+    myLaptop.hardDriveDetails();
     FishTank aquarium;
-    aquarium.tempAlert();
+    aquarium.printTemp();
+    std::cout << aquarium.waterTempCelcius << std::endl;
+    std::cout << aquarium.gallons << std::endl;
+    FishTank fishtemp;
+    fishtemp.tempAlert();
+    std::cout << fishtemp.waterTempCelcius << std::endl;
+    MidiKeyboard pitchMove;
+    pitchMove.pitchShift();
+    
+    MusicStudio studio;
+    StudioLighting lighting;
```

Comment:
These 2 lines are causing linker errors.  
you need to implement the constructors for these 2 classes
- [ ] revise